packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -136,15 +136,27 @@ module.exports = {
        * typically include will not be specified that way
        *
        * @type {{
-          extends: string[];
+          extends: string;
           include: string[];
-          excludes: string[];
+          exclude: string[];
           compilerOptions: Record<string,unknown>;
           references?: Array<{path:string}>
           }}
        */
     const tsconfig = jju.parse(fs.readFileSync(tsconfigPath).toString());
 
+    /**
+     * Handle any necessary extends merging here, make sure to treat just like native tsconfigs would
+     */
+    if (tsconfig.extends) {
```

Comment:
This will only handle one level of extends. A more complete fix would be to use typescript's tsconfig parsing methods like this. (The one weird caveat is that it's probably best to do `const ts = require('typescript')` within this method rather than at the top of the file to avoid unconditionally requiring/parsing all of typescript any time the package is referenced.)
https://github.com/microsoft/fluentui/blob/c9bd5b5a02ae94a8239ce1b56cac2f0d0dbd3586/scripts/gulp/plugins/util/docgen.ts#L157-L169

Reply:
we considered that at one point, wasn't sure if there was a reason that jju was used in the first place

Reply:
reverting to jju approach as adding ts and the ts methods add a ton of overhead that aren't needed to read a value out of a json file

Reply:
Good job TS 🤦‍♀️